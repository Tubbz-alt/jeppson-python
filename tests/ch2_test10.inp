   0.33333   0.5     150.0      1.217E-5    7.0E-6  32.2    
