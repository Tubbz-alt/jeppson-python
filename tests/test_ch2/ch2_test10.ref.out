Q=    0.5000D=    0.3333L=    150.00F=   0.01655HEADLOSS=    3.7976
